samples/cpp/benchmark_app/utils.hpp

```diff
@@ -149,3 +154,122 @@ void convert_io_names_in_map(
     }
     map = new_map;
 }
+
+void verifyBinaryFile(std::ifstream& binaryFile, const std::string& fileName, const unsigned long inputSize);
+template <typename T>
+void verifyNumpyFile(std::ifstream& binaryFile,
+                     const std::string& fileName,
+                     const ov::Shape& inputShape,
+                     const unsigned long inputSize) {
+    auto fullFileSize = static_cast<std::size_t>(binaryFile.tellg());
+    binaryFile.seekg(0, std::ios_base::beg);
+    OPENVINO_ASSERT(binaryFile.good(), "Can not read ", fileName);
+
+    std::string magic_string(6, ' ');
+    binaryFile.read(&magic_string[0], magic_string.size());
+    OPENVINO_ASSERT(magic_string == "\x93NUMPY",
+                    "Numpy file has incorrect magic string. File ",
+                    fileName,
+                    " might be corrupted");
+
+    binaryFile.ignore(2);  // Skip major and minor version bytes
+    unsigned short headerSize;
```

Comment:
Are you sure `short` is the correct type? Its size may vary depending on a platform

Reply:
I got it from https://numpy.org/doc/1.13/neps/npy-format.html - it states:
 `The next 2 bytes form a little-endian unsigned short int: the length of the header data HEADER_LEN.`